hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java

```diff
@@ -1190,49 +1195,6 @@ public void run() {
     }
 
     @Test
-    public void testNearCacheTTLCleanup_triggeredViaPut() {
```

Comment:
Expiration should *not* be triggered via put, just eviction. So this test didn't work out anymore with checking the expiration/eviction stats accordingly.